app/controllers/agenda-items/agenda-item.ts

```diff
@@ -12,6 +12,10 @@ export default class AgendaItemController extends Controller {
     return !!this.model.vote;
   }
 
+  get wasHandled() {
+    return Boolean(this.model.agendaItem.belongsTo('handledBy').value());
```

Comment:
This assumes the relationship is loaded (which is why I added the include). We _could_ do it from the template but it would require `{{#let}}` and some sort of `await` helper combo that wraps all the places where we need it. I prefer this approach myself since it makes the transition to sync relationships easier (which I still think is the best setup).